languages/elixir/exercises/concept/tuples/.docs/after.md

```diff
@@ -0,0 +1,74 @@
+Great job! [Tuples][tuple-doc] are used commonly to group information informally. A common pattern through elixir is to group function return values with a status
+
+```elixir
+File.read("hello.txt")
+# => {:ok, "World"}
+
+File.read("invalid.txt")
+# => {:error, :enoent}
+```
+
+Then when writing elixir functions, we can made use of an [assertive style][assertive-style] with [pattern matching][pattern-match-doc]:
+
+```elixir
+def read_file() do
+  {:ok, contents} = File.read("hello.txt")
+  contents
+end
+```
+
+It might occur to you that this function may crash if the file does not exist. Don't worry, in Elixir it is often said to [**let it crash**][let-it-crash], because in elixir applications, a supervising process will restart the application to a known-good state.
+
+## Key-points
+
+### [Tuples][tuple-doc]
+
+- [Tuple][tuple-doc] literals are enclosed with curly braces, `{}`.
+- [Tuples][tuple-doc] may hold any data-type in contiguous memory.
+- When manipulating a [tuple][tuple-doc], rather than mutating the existing [tuple][tuple-doc], a new one is created.
```

Comment:
@neenjaw I'm working on https://github.com/exercism/v3/pull/2377 and I noticed that there is no link named `tuple-doc`. Any idea what did you mean to link here?

Reply:
probably just elixir's tuple documentation from the getting started or official docs